backend/pkg/scheduler.go

```diff
@@ -1,14 +1,54 @@
 package main
 
-import "github.com/grafana/grafana-plugin-sdk-go/backend/log"
+import (
+	"fmt"
+
+	"github.com/360EntSecGroup-Skylar/excelize"
+	"github.com/grafana/grafana-plugin-sdk-go/backend/log"
+	"gopkg.in/gomail.v2"
+)
 
 type EmailConfig struct {
 	email    string
 	password string
 }
 
+type SmtpServer struct {
+	host string
+	port string
+}
```

Comment:
Possibly should have added this to previous review, but another minor.

Think convention in go to define structs with factory creators (kind of like the equivalent of constructors). E.g.

```
func newEmailConfig(email string, password string) *EmailConfig {
  c := EmailConfig{email: email, password: password};
  return &c;
}
```

Pretty redundant here, but think it's just a good pattern to follow (imagine idea is that it starts becoming useful for more complex structs which have constructor logic and defaults etc.).

Reply:
Cool - have looked into this and have found the convention to be a func `New` is there is only a single exposed type in a module otherwise `NewTypeX`